Examples/Python/tests/helpers/__init__.py

```diff
@@ -5,6 +5,11 @@
 from acts.examples import BareAlgorithm
 
 geant4Enabled = any(v.startswith("G4") for v in os.environ.keys())
+if geant4Enabled:
+    try:
+        import acts.examples.geant4
+    except ImportError:
+        geant4Enabled = False
```

Comment:
I added this to get the pytest to work. It gave errors when DD4HEP was enabled but not Geant4. Is this OK, @paulgessinger? I don't know why `geant4Enabled` is handled differently from `dd4hepEnabled`.

Reply:
The import check checks if the module is importable. I don't remember exactly what I was thinking, but the other check checks if the user has sourced the G4 file, meaning that the necessary environment variables are set. I guess I assumed if that is the case, the import should be safe to do.

Adding this extra check is probably fine!